### PR TITLE
Rw test doc strings

### PR DESCRIPTION
**Pull request by rtwhitfield84**
Added Docstrings to conform with coding standard of the squids

## Related Issue
Coding standards

## Motivation and Context
To keep up with coding standards

## Types of changes
 What types of changes does your code introduce? Put an \`x\` in all the boxes that apply: 
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
 Go over all the following points, and put an \`x\` in all the boxes that apply. 
 If you're unsure about any of these, don't hesitate to ask. We're here to help! 
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by nchemsak — APPROVED** [review]
👍

File: TestChooseActiveCustomer.py [hunk 1]
```diff
@@ -4,6 +4,19 @@
 
 class TestChooseActiveCustomer(unittest.TestCase):
```
Comment (asimonia) [inline TestChooseActiveCustomer.py:6]: I like this 👍

File: superuser.py [hunk 2]
```diff
@@ -2,6 +2,22 @@
 
 class Superuser():
 
+    '''
```
Comment (asimonia) [inline superuser.py:5]: 💯

**Review by asimonia — APPROVED** [review]
Good

**Pull request merged**